docs/pages/get-started/create-a-project.mdx

```diff
@@ -1,117 +1,22 @@
 ---
 title: Create a project
-description: Learn how to create a new Expo project and run it on your device.
+description: Learn how to create a new Expo project.
+hideTOC: true
 ---
 
-import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
-import ImageSpotlight from '~/components/plugins/ImageSpotlight';
-import { Step } from '~/ui/components/Step';
-import { BoxLink } from '~/ui/components/BoxLink';
-import { BookOpen02Icon, GraduationHat02Icon } from '@expo/styleguide-icons';
 
-The process of running a new Expo project consists of three steps. You start by initializing a new project, then start the development server with Expo CLI and finally open the app on your device with Expo Go to see your changes live.
+System requirements:
 
-<Step label="1">
+- [Node.js (LTS)](https://nodejs.org/en/).
+- macOS, Windows (Powershell and WSL 2), and Linux are supported.
 
-## Initialize a new project
+We recommend creating a project with the default project template, which includes example code to help get you started. To create a new project, run the following command:
 
-To initialize a new project, use [`create-expo-app`](/more/glossary-of-terms/#create-expo-app) to run the following command:
+<Terminal cmd={['$ npx create-expo-app@latest']} />
 
-<Terminal
-  cmd={[
-    '# Create a project named my-app',
-    '$ npx create-expo-app my-app',
-    '',
-    '# Navigate to the project directory',
-    '$ cd my-app',
-  ]}
-  cmdCopy="npx create-expo-app my-app && cd my-app"
-/>
+> You can use another template by adding the `--template` flag.
```

Comment:
I am rewording to this:

```
> You can select a different template by adding the `--template` flag.
```

- Implicitly, different templates are available. I don't think we need to say "available"
- I liked "select" from your rendition, so I took that.
- I said "different" instead of "from other"

With that, please lmk if you disagree and want another wording.